content/waku/milestones/open/2025-explore-peer-discovery-gap.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Explore Peer Discovery Gap
+tags:
+    - waku-milestone
+date: 2025-02-10
+---
+
+Github Milestone: https://github.com/waku-org/pm/milestone/44
+
+The current peer discovery protocols, discv5, is limited in regards to finding peers with specific characteristics (shards, mounted protocols, etc). As such characteristics are filtered after the fact, and not part of the random walk mechanism.
+
+This implementation impedes the decentralization of store services and implementation of a mixnet.
+
+In this milestone, we will implement PoCs for those two features, to highlight the caveats of discv5 and define requirements for an upgrade of the peer discovery for Waku.
+
+### Deliverable: [Waku Sync]()
```

Comment:
old deliverable can be re-used. I've renamed it.https://github.com/waku-org/pm/issues/132